administration/src/components/generation/PdfFactory.ts

```diff
@@ -5,6 +5,34 @@ import {CardActivateModel} from "../../generated/compiled";
 import uint8ArrayToBase64 from "../../util/uint8ArrayToBase64";
 import {format, fromUnixTime} from "date-fns";
 import {getRegions_regions as Region} from "../../graphql/regions/__generated__/getRegions";
+import {Err, Ok, Result} from "../../util/result";
+
+export type PDFError = UnexpectedUnicodeError | PDFGenerationError
+
+type PDFGenerationError = {
```

Comment:
Is this used anywhere? Perhaps you should wrap the whole thing in a try catch and return this err on catch?

Reply:
I think this is worth discussing. There can always be unexpected crashes, but I dont think we should just add try catches for errors we do not expect.

Reply:
But then this is currently unused and could/should be removed, right?

Reply:
oh sorry, yes, I forgot to use this error type. I planned to use it around the .output() function :) thanks!